hphp/tools/oss_hot_section_ordering

```diff
@@ -818,7 +818,7 @@
 .text.hash_string_i_unaligned_crc*
 .text._ZN4HPHP9ArrayData10IsValidKeyERKNS_7VariantE*
 .text._ZN4HPHP21tvMatchesRepoAuthTypeENS_10TypedValueENS_12RepoAuthTypeE*
-.text._ZN4HPHP18IsSSEHashSupportedEv*
+.text._ZN4HPHP17IsHwHashSupportedEv*
```

Comment:
How is this file generated? @binliu19 The hash-crc-x74.S assembly has the macro IF_SSE42() to define the function names, but those names didn't make it into this file? Should I add them? How?